test/functional/api/v2/discovered_hosts_controller_test.rb

```diff
@@ -29,6 +29,9 @@ def switch_controller(klass)
                        :name => 'discovery_reboot',
                        :value => true,
                        :category => 'Setting::Discovered')
+
+    #load the power.rb file, as PowerService is not autoloaded from file named power
+    ForemanDiscovery::NodeAPI::Power 
```

Comment:
that file has a few classes defined in it, renaming the file isn't going to do much, but maybe we should split it to one class per file. don't think it's important enough, but if you'd like, I'll do it.